sapi/fpm/tests/tester.inc

```diff
@@ -627,7 +627,7 @@ class Tester
         $read = [$this->outDesc];
         $write = null;
         $except = null;
-        if (stream_select($read, $write, $except, 2 )) {
+        if (stream_select($read, $write, $except, 3)) {
```

Comment:
Why does it need to be increased?

Reply:
The test case waits one second after sigquit to send sigterm and another second after that to send sigkill, so we need to wait for two seconds for the reload to occur. So the timeout should be larger than two seconds.